go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -189,13 +189,13 @@ func runOvnKube(ctx *cli.Context) error {
 		return fmt.Errorf("failed to initialize exec helper: %v", err)
 	}
 
-	clientset, err := util.NewClientset(&config.Kubernetes)
+	clientset, egressFirewallClientset, err := util.NewClientsets(&config.Kubernetes)
```

Comment:
as with the comments I made on [the egress IP bug](https://github.com/ovn-org/ovn-kubernetes/pull/1484#pullrequestreview-457535277) this doesn't seem like a very clean / future-feature-friendly way of doing this. (And this PR is going to end up conflicting with the egress IP one here).

Though in contrast to that PR, this one does a much better job of keeping the egress-firewall-specific controller code separate from the generic controller code